### PR TITLE
HW1 Submission Jue Zhou

### PR DESCRIPTION
**Pull request by Jade-Zhou**
First I forked the hw01 repo from the cfss page and cloned it to my Rstudio. I pulled from Github to start my work. I wrote the self-introduction using markdown (https://rstudio.com/wp-content/uploads/2016/03/rmarkdown-cheatsheet-2.0.pdf, this cheatsheet is rather useful.) 
Adding an image in took some time. I tried it locally by adding an image to my files, but it didn't show up on the website. The TA helped me to upload it, but it still didn't show up. I changed the name of the image (from "a comic.jpg" to "comic.jpg"), and it finally worked. Anyway, the readme seems much better now.

### CODE REVIEW

**Review by yanwenwang24 — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    *       |              |            |
| **Coding strategy**         |    *       |              |            |
| **Presentation**            |     *      |              |            |
| **Achievement, creativity** |    *      |              |            |
| **Ease of access**          |    *       |              |            |

Excellent!

File: README.md [hunk 1]
```diff
@@ -1 +1,17 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## A brief introduction of me
```
Comment (bensoltoff) [inline README.md:3]: Good practice is to add a line break between headers and text content. Will avoid formatting issues in the future

File: README.md [hunk 2]
```diff
@@ -1 +1,17 @@
...
+My name is *Jue Zhou* (The "e" is pronounced here as /e/. You could also call me Jade because my name means Jade in Chinese.) 
+
+I come from southeast China and I did my undergrad at Peking University. Now I am a MAPSS student at Uchicago and I am in sociology concentration. My research interests generally lie in the area of gender and politics. To be more specific, I focus on the changes of gender roles in the rising of nation states. 
+
+* My hobbies
+  + reading novels (Jane Austin, Eileen Chang and Ursula K. LeGuin)
+  + watching movies (My favorite director is Kieślowski.)
+  + visiting museums
+
+Welcome to douban.com (a Chinese website for people to mark and review books, movies, TV series, video games) to share your favorite books, movies and games with me. This is my personal page: <https://www.douban.com/people/92626129/>
```
Comment (bensoltoff) [inline README.md:13]: A more clean appearing link:

```
[This is my personal page](https://www.douban.com/people/92626129/)
```

[This is my personal page](https://www.douban.com/people/92626129/)

File: README.md [hunk 3]
```diff
@@ -1 +1,17 @@
...
+
+* My hobbies
+  + reading novels (Jane Austin, Eileen Chang and Ursula K. LeGuin)
+  + watching movies (My favorite director is Kieślowski.)
+  + visiting museums
+
+Welcome to douban.com (a Chinese website for people to mark and review books, movies, TV series, video games) to share your favorite books, movies and games with me. This is my personal page: <https://www.douban.com/people/92626129/> 
+
+PS: Here is a comic book about PhD life. It's very funny. Go and check it!
+![Image](comic.jpg)
```
Comment (bensoltoff) [inline README.md:16]: Nice addition of an image in the repo and embedding it in the document 👍

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |        x      |            |
| **Coding strategy**         |           |      x        |            |
| **Presentation**            |        x   |              |            |
| **Achievement, creativity** |   x        |              |            |
| **Ease of access**          |      x     |              |            |

**Pull request closed without merging**